impl/baseline/runner.ts

```diff
@@ -49,12 +51,59 @@ function test(request: ConformanceRequest): ConformanceResponse["result"] {
       value: toBinary(FailureSetSchema, failureSet),
     };
   }
+  // Returning a runtime error for the test Required.Proto3.ProtobufInput.UnknownOrdering.ProtobufOutput
+  // crashes the runner.
+  if (
+    is_Required_Proto3_ProtobufInput_UnknownOrdering_ProtobufOutput(request)
+  ) {
+    return {
+      case: "protobufPayload",
+      value: new Uint8Array(),
+    };
+  }
```

Comment:
This is a bug in the conformance runner. The test protocol allows clients to reply with a runtime error, which is considered a test failure. This specific test however does not allow the client to do so, and prematurely exits the test suite, making it impossible to skip or mark the test as known failing.